cookbooks/aws-parallelcluster-common/spec/unit/resources/efa_spec.rb

```diff
@@ -146,6 +147,25 @@ def mock_efa_supported(supported)
       end
     end
   end
+
+  context 'when rhel version is older than 8.4' do
+    let(:chef_run) do
+      ChefSpec::Runner.new(
+        # Create a runner for the given platform/version
+        platform: "redhat",
```

Comment:
If it is possible to use the `node['platform_version']`in the code above, in the ChefSpec `version` should be able to override that value

Reply:
Nope, I tried that, it does not work because there is no platform version that can be used for the specific red hat version.

Error is `Could not find platform 'redhat/8.3' `